Dockerfile

```diff
@@ -1,31 +1,45 @@
-FROM alpine:3.13 AS build-base
-RUN apk add git make cmake g++ libusb-dev libpulse
+FROM alpine:latest AS build-base
+RUN apk add --no-cache git make cmake g++ libusb-dev libpulse
 
 FROM build-base AS rtl_fm
-ARG RTL_SDR_VERSION=0.6.0
-RUN git clone --depth 1 --branch ${RTL_SDR_VERSION} https://github.com/osmocom/rtl-sdr.git /opt/rtl_sdr
+RUN git clone --depth 1 https://gitea.osmocom.org/sdr/rtl-sdr.git /opt/rtl_sdr
 WORKDIR /opt/rtl_sdr/build
 RUN cmake .. && make
+RUN make install
 
 FROM build-base AS multimon
-ARG MULTIMON_VERSION=1.1.9
-RUN git clone --depth 1 --branch ${MULTIMON_VERSION} https://github.com/EliasOenal/multimon-ng.git /opt/multimon
+RUN git clone --depth 1 https://github.com/EliasOenal/multimon-ng.git /opt/multimon
 WORKDIR /opt/multimon/build
 RUN cmake .. && make
 
-FROM alpine:3.13 AS boswatch
+FROM alpine:latest AS boswatch
 ARG BW_VERSION=develop
 RUN apk add git && \
     git clone --depth 1 --branch ${BW_VERSION} https://github.com/BOSWatch/BW3-Core.git /opt/boswatch
 
 
-FROM python:3.9.1-alpine AS runner
+FROM python:alpine AS client
 LABEL maintainer="bastian@schroll-software.de"
```

Comment:
ist inzwischen `info@schroll-it.de`
Wenn du magst, kannst du dich auch dazuschreiben.